search/search-dev-tools/src/style/prism.scss

```diff
@@ -28,7 +28,7 @@
 	}
 
 	pre > code[class*='language-'] {
-		font-size: 1em;
+		// font-size: 1em;
```

Comment:
I would suggest to remove the entire CSS rule :-)

Reply:
thanks, sloppy Friday afternoon coding right here :)